ext/opcache/tests/bug80900.phpt

```diff
@@ -0,0 +1,49 @@
+--TEST--
+Bug 80900: Switch constant with incorrect type
+--INI--
+opcache.enable=1
+opcache.enable_cli=1
+opcache.optimization_level=-1
+opcache.opt_debug_level=0x20000
```

Comment:
Personally, for miscompiles I would only test program behavior, not produced opcodes. Opcodes are less stable, and only needed if we need to check that an optimization happens, not that it retains the same behavior.

Reply:
You can already see this issue with the failure on the I386 build, where the stored function stack size differs...

Reply:
I see, thanks!